generators/server/templates/gradlew

```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS='"-Xmx64m"'
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
Cann't this work with `"-Xmx64m -Xms64m"`, without adding many quotes?

Reply:
This file is part of the gradle distribution, so we don't have any influence on how they ship this file.